test/DynamoCoreTests/SchedulerTests.cs

```diff
@@ -1016,6 +1022,12 @@ public void TestTaskQueuePreProcessing00()
                 MakeAggregateRenderPackageAsyncTask(Guid.Empty),
             };
 
+            // Due to defaulting to auto-run mode, multiple UpdateGraphAsyncTask
+            // may have been scheduled prior to this. Clear those tasks before test 
+            // starts in a predictable state.
+            // 
+            schedulerThread.GetSchedulerToProcessTasks();
+
```

Comment:
Hi @ke-yu, when we get here, the `DynamoModel.Scheduler` already contains 3 instances of `UpdateGraphAsyncTask` objects in its queue due to auto-run. So I'm clearing the task through `ISchedulerThread` implementation.